a8c-files.php

```diff
@@ -19,6 +19,9 @@
 
 define( 'ALLOW_UNFILTERED_UPLOADS', false );
 
+// See inline for usage of this constant
+define( 'WPCOM_VIP_IMAGE_QUALITY_100', 'allow-quality-100' );
```

Comment:
As discussed, there was some confusion in how this is supposed to be used. My comment below clarifies usage: https://github.com/Automattic/vip-go-mu-plugins/pull/513#issuecomment-286887640